pkg/agent/apiserver/handlers/ovstables/handler.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ovstables
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"k8s.io/klog/v2"
+
+	agentquerier "antrea.io/antrea/pkg/agent/querier"
+)
+
+type Response struct {
+	Id   string `json:"id,omitempty"`
+	Name string `json:"name,omitempty"`
+}
+
+func HandleFunc(aq agentquerier.AgentQuerier) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		tableStrs, err := aq.GetOVSCtlClient().DumpTables()
+		if err != nil {
+			klog.Errorf("Failed to dump tables: %v", err)
```

Comment:
switch to `klog.ErrorS` (same in other places)

Reply:
Done.